RaisinToast.podspec

```diff
@@ -0,0 +1,35 @@
+#
+# Be sure to run `pod lib lint RaisinToast.podspec' to ensure this is a
+# valid spec and remove all comments before submitting the spec.
+#
+# Any lines starting with a # are optional, but encouraged
+#
+# To learn more about a Podspec see http://guides.cocoapods.org/syntax/podspec.html
+#
+
+Pod::Spec.new do |s|
+  s.name             = "RaisinToast"
+  s.version          = "1.0.0"
+  s.summary          = "A UIWindow subclass used to message information to the users of your app."
+  s.description      = <<-DESC
+                       Provides a messaging window layer and a default "toast" view controller, ideal for presenting errors, warnings and feedback throughout your app.
+                       DESC
+  s.homepage         = "https://github.com/Raizlabs/RaisinToast"
+  # s.screenshots     = "www.example.com/screenshots_1", "www.example.com/screenshots_2"
```

Comment:
Hmm I know I made these same comments on an identical podspec, but that was `Example/Pods/Local Podspecs/RaisinToast.podspec`. Those comments should be for this main one